projects/forge/src/main/java/dan200/computercraft/ComputerCraft.java

```diff
@@ -95,6 +96,7 @@ private static void syncConfig(ModConfig config) {
         if (!config.getModId().equals(ComputerCraftAPI.MOD_ID)) return;
 
         if (config.getType() == ModConfig.Type.SERVER && ((ForgeConfigFile) ConfigSpec.serverSpec).spec().isLoaded()) {
+            ProxyPasswordConfig.init(config.getFullPath().resolveSibling(ComputerCraftAPI.MOD_ID + "-proxy.pw"));
```

Comment:
Could we pass the config path into `ConfigSpec.syncServer`, and then call ` ProxyPasswordConfig.init` in there instead?

Also, do you know what happens when a client connects to a dedicated server here? I fear this will crash, as the path is only available on the server. I wonder if we need something like `config.getConfigData() instanceof FileConfig fileConfig ? fileConfig.getNioPath() : null`.